parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -201,6 +238,33 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type PalletOperatingMode<T: Config> = StorageValue<_, BasicOperatingMode, ValueQuery>;
 
+	/// The base fee to cover the cost of submitting an outbound message
+	/// default 0.1 DOT
+	#[pallet::type_value]
+	pub fn DefaultBaseFee() -> u128 {
+		1_000_000_000
+	}
+	#[pallet::storage]
+	pub type BaseFee<T: Config> = StorageValue<_, u128, ValueQuery, DefaultBaseFee>;
+
+	/// The swap ratio from Ether->DOT from https://www.coingecko.com/en/coins/polkadot/eth
+	/// default 1:400
+	#[pallet::type_value]
+	pub fn DefaultSwapRatio() -> u128 {
+		400
+	}
+	#[pallet::storage]
+	pub type SwapRatio<T: Config> = StorageValue<_, u128, ValueQuery, DefaultSwapRatio>;
+
+	/// The gas price to calculate the reward in Ether from https://etherscan.io/gastracker
+	/// default 15 gwei
+	#[pallet::type_value]
+	pub fn DefaultGasPrice() -> u128 {
+		15_000_000_000
+	}
+	#[pallet::storage]
+	pub type GasPrice<T: Config> = StorageValue<_, u128, ValueQuery, DefaultGasPrice>;
+
```

Comment:
I think we're adding so many fee-related storage items here that we should rather them group them into a single struct. Then we only need a single getter/setter pair.

Reply:
https://github.com/Snowfork/snowbridge/pull/940/commits/3b784b20f4526bd31023e1279efe85d27613d1b5